game/quake2_md2.ksy

```diff
@@ -0,0 +1,429 @@
+        [-0.688191, -0.587785, 0.425325], 
+        [-0.681718, -0.147621, 0.716567], 
+        [-0.442863, -0.238856, 0.864188], 
+        [-0.587785, -0.425325, 0.688191], 
+        [-0.309017, -0.500000, 0.809017], 
+        [-0.147621, -0.716567, 0.681718], 
+        [-0.425325, -0.688191, 0.587785], 
+        [-0.162460, -0.262866, 0.951056], 
+        [0.442863, -0.238856, 0.864188], 
+        [0.162460, -0.262866, 0.951056], 
+        [0.309017, -0.500000, 0.809017], 
+        [0.147621, -0.716567, 0.681718], 
+        [0.000000, -0.525731, 0.850651], 
+        [0.425325, -0.688191, 0.587785], 
+        [0.587785, -0.425325, 0.688191], 
+        [0.688191, -0.587785, 0.425325], 
+        [-0.955423, 0.295242, 0.000000], 
+        [-0.951056, 0.162460, 0.262866], 
+        [-1.000000, 0.000000, 0.000000], 
+        [-0.850651, 0.000000, 0.525731], 
+        [-0.955423, -0.295242, 0.000000], 
+        [-0.951056, -0.162460, 0.262866], 
+        [-0.864188, 0.442863, -0.238856], 
+        [-0.951056, 0.162460, -0.262866], 
+        [-0.809017, 0.309017, -0.500000], 
+        [-0.864188, -0.442863, -0.238856], 
+        [-0.951056, -0.162460, -0.262866], 
+        [-0.809017, -0.309017, -0.500000], 
+        [-0.681718, 0.147621, -0.716567], 
+        [-0.681718, -0.147621, -0.716567], 
+        [-0.850651, 0.000000, -0.525731], 
+        [-0.688191, 0.587785, -0.425325], 
+        [-0.587785, 0.425325, -0.688191], 
+        [-0.425325, 0.688191, -0.587785], 
+        [-0.425325, -0.688191, -0.587785], 
+        [-0.587785, -0.425325, -0.688191], 
+        [-0.688191, -0.587785, -0.425325] ]
+types:
+  tex_point:
+    seq:
+      - id: u_px
+        type: u2
+      - id: v_px
+        type: u2
+    instances:
+      u_normalized:
+        value: (u_px + 0.0) / _root.skin_width_px
+      v_normalized:
+        value: (v_px + 0.0) / _root.skin_height_px
+  triangle:
+    seq:
+      - id: vertex_indices
+        type: u2
+        repeat: expr
+        repeat-expr: 3
+        doc: indices to `_root.frames[i].vertices` (for each frame with index `i`)
+      - id: tex_point_indices
+        type: u2
+        repeat: expr
+        repeat-expr: 3
+        doc: indices to `_root.tex_coords`
+  vec3f:
+    seq:
+      - id: x
+        type: f4
+      - id: y
+        type: f4
+      - id: z
+        type: f4
+  compressed_vec:
+    seq:
+      - id: x_compressed
+        type: u1
+      - id: y_compressed
+        type: u1
+      - id: z_compressed
+        type: u1
+    instances:
+      x:
+        value: x_compressed * _parent._parent.scale.x + _parent._parent.translate.x
+      y:
+        value: y_compressed * _parent._parent.scale.y + _parent._parent.translate.y
+      z:
+        value: z_compressed * _parent._parent.scale.z + _parent._parent.translate.z
+  vertex:
+    seq:
+      - id: position
+        type: compressed_vec
+      - id: normal_index
+        type: u1
+        doc: |
+          `normal = anorms_table[normal_index]`
```

Comment:
Given that you included the `anorms_table` definition, you can actually resolve the `normal` vector now:

```suggestion
      - id: normal_index
        type: u1
    instances:
      normal:
        value: _root.anorms_table[normal_index]
```

Reply:
Excellent! I was hoping this was possible.